model_analyzer/record/metrics_manager.py

```diff
@@ -117,7 +117,7 @@ def _init_state(self):
             if self._gpus[i].device_uuid() not in gpu_info:
                 device_info = {}
                 device = numba.cuda.list_devices()[i]
-                device_info['name'] = device.name
+                device_info['name'] = str(device.name, encoding='utf-8')
```

Comment:
why was this needed?

Reply:
The device name is returned as a byte-stream - this wasn't an issue before because it was stored in the checkpoint and then read back (which automatically converted it to a string when it went into json format).